src/skinview-overrides.ts

```diff
@@ -0,0 +1,34 @@
+
+import { TextureCanvas, TextureSource } from "skinview-utils";
+
+function computeCapeScale(image: TextureSource): number {
+	if (image.width === 2 * image.height) {
+		// 64x32
+		return image.width / 64;
+	} else if (image.width * 17 === image.height * 22) {
+		// 22x17
+		return image.width / 22;
+	} else if (image.width * 11 === image.height * 23) {
+		// 46x22
+		return image.width / 46;
+    } else if ((image.height % (image.width / 2)) == 0) {
+		//Animated cape HD with raito 64x32
```

Comment:
nit: a space before starting comment text

Reply:
Moved to https://github.com/bs-community/skinview-utils/pull/15